server/views/applications/confirm.njk

```diff
@@ -4,40 +4,33 @@
 
 {% extends "../partials/layout.njk" %}
 
-{% set pageTitle = applicationName + " - " + pageHeading %}
+{% set pageTitle = applicationName + " - Referral complete" %}
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block content %}
 
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            {{ govukPanel({
-                titleText: "Application confirmation"
-            }) }}
-            <p>
-          We’ve sent you a confirmation email.
-        </p>
-            <p>
-                <a href="">Download a copy of this application (PDF, 708 KB) </a>
-            </p>
-
-            <h2 class="govuk-heading-m">What happens next</h2>
-
-            <p>We’ve sent your application to the Midlands central referral unit (CRU) for Approved Premises (AP).</p>
-            <p>The Midlands CRU will review your application within 10 working days.</p>
-            <p>You'll receive an email when the status of your application changes, or you can check the status in your <a href="">AP applications dashboard.</a>
-            </p>
-            <p>If you could not provide a release date you will need to email the Midlands CRU as soon as this date is confirmed.</p>
-            <p>
-                <a href="https://www.gov.uk/service-manual/user-centred-design/resources/patterns/feedback-pages.html">What did you think of this service?</a> (takes 30 seconds)</p>
-
-        </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukPanel({
+        titleText: "Referral complete"
+      }) }}
+      <p>We’ve sent you a confirmation email.</p>
```

Comment:
Is this true? I wasn't aware that CAS1 were sending confirmation emails. As far as I know the Notify integration has yet to be merged into the API yet?

Reply:
The dynamic page headings - it's just not really a pattern we've followed in CAS3. You either have the heading in the njk, or you have it in the controller, and on CAS3 we've gone for njk. Just wanted to keep things consistent while I was revisiting these files